src/test/scala/nl.knaw.dans.easy.bagstore/component/BagStoresSpec.scala

```diff
@@ -139,8 +140,8 @@ class BagStoresSpec extends TestSupportFixture
   }
 
   it should "return empty stream if BagStore is empty" in {
-    inside(bagStores.enumBags().map(_.toList)) {
-      case Success(bagIds) => bagIds shouldBe empty
+    bagStores.enumBags().map(_.toList) should matchPattern {
+      case Success(bagIds: List[_]) if bagIds.isEmpty  =>
```

Comment:
is this `: List[_]` check actually necessary? Isn't that what the type already says?

Reply:
the isEmpty part does not compile, if i don't mention the type for some reason.